### PR TITLE
En_Kz OK

### PR DESCRIPTION
**Pull request by fig02**
mweep

### CODE REVIEW

File: src/overlays/actors/ovl_En_Kz/z_en_kz.c [hunk 1]
```diff
@@ -21,47 +28,458 @@ const ActorInit En_Kz_InitVars = {
...
+    Vec3f initPos;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Kz/func_80A9CE44.s")
+    pos = this->actor.posRot.pos;
+    initPos = this->actor.initPosRot.pos;
+    pos.y += 60.0f;
+    initPos.y += -100.0f;
+    initPos.z += 260.0f;
+    func_800C04D8(globalCtx, this->cutsceneCamera, &pos, &initPos);
+    if (EnKz_FollowPath(this, globalCtx) == 1 && (this->waypoint == 0)) {
```
Comment (ethteck) [inline src/overlays/actors/ovl_En_Kz/z_en_kz.c:374]: I think we want this condition to have parens around it like `(EnKz_FollowPath(this, globalCtx) == 1)`?

File: src/overlays/actors/ovl_En_Kz/z_en_kz.c [hunk 2]
```diff
@@ -21,47 +28,458 @@ const ActorInit En_Kz_InitVars = {
...
+    }
+    this->actionFunc(this, globalCtx);
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Kz/func_80A9D490.s")
+s32 EnKz_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    s32 limb = limbIndex;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Kz/func_80A9D520.s")
+    if (limb == 8 || limb == 9 || limb == 10) {
```
Comment (ethteck) [inline src/overlays/actors/ovl_En_Kz/z_en_kz.c:452]: same thing about parens around conditions

File: src/overlays/actors/ovl_En_Kz/z_en_kz.c [hunk 3]
```diff
@@ -21,47 +28,458 @@ const ActorInit En_Kz_InitVars = {
...
+        rot->y += (Math_Sins(THIS->unk_2A6[limb]) * 200.0f);
```
Comment (ethteck) [inline src/overlays/actors/ovl_En_Kz/z_en_kz.c:453]: conversely, I don't think this needs parens around the entire thing

File: src/overlays/actors/ovl_En_Kz/z_en_kz.c (hunk 2 above)
Comment (fig02) [reply src/overlays/actors/ovl_En_Kz/z_en_kz.c:452]: so far we only use them when it makes sense to for readability. in this case it looks just fine without imo, and id say that adding them might make it even harder to read. i agree with the other two though

File: src/overlays/actors/ovl_En_Kz/z_en_kz.c [hunk 4]
```diff
@@ -21,47 +28,458 @@ const ActorInit En_Kz_InitVars = {
...
+
+    return 0;
+}
+
+void func_80A9CB18(EnKz* this, GlobalContext* globalCtx) {
+    Player* player = PLAYER;
+
+    if (func_80A9C95C(globalCtx, this, &this->unk_1E0.unk_00, 340.0f, EnKz_GetText, func_80A9C6C0) != 0) {
+        if (this->actor.textId == 0x401A) {
+            if (!(gSaveContext.eventChkInf[3] & 8)) {
```
Comment (fig02) [inline src/overlays/actors/ovl_En_Kz/z_en_kz.c:217]: reminder to myself to and these together tomorrow

File: src/overlays/actors/ovl_En_Kz/z_en_kz.c (hunk 2 above)
Comment (ethteck) [reply src/overlays/actors/ovl_En_Kz/z_en_kz.c:452]: Oh, really? I thought it was an "always" sort of thing. I'm much happier with it being not that way though.

**Pull request merged**